config/samples/kustomization.yaml

```diff
@@ -4,5 +4,5 @@
 resources:
 # - discovery_v1_discoveredcluster.yaml
 - discovery_v1_discoveryconfig.yaml
-- discovery_v1_discoveredclusterrefresh.yaml
+# - discovery_v1_discoveredclusterrefresh.yaml
```

Comment:
Why comment this out? Unnecessary?

Reply:
Yeah the cluster refresh isn't necessary when calling the `make samples` command because the act of creating the discoveryconfig is enough to trigger a reconcile in the operator

Reply:
And if the refresh is created before the config it causes an error since the config can't be found yet